04week/nasa/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>NASA website</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header>
+    <h1>NASA
+    </h1>
+  </header>
+<div id="container">
+  <div class="column">
+    <h3>Social Media</h3>
+    <p>
+      Twitter - @nasa<br>
+      Facebook - fb.com/nasa<br>
+      Instagram - @nasa<br>
+      Youtube - youtube.com/NASA<br>
+    </p>
+  </div>
+  <div class="column">
+    <h3>OFFICE INFO</h3>
+    <p>
+      Public Communications Office<br>
```

Comment:
I would have done each line as a separate element so you can have more control over spacing and padding.